zad16.py

```diff
@@ -0,0 +1,39 @@
+from meszstack import meszgen
+
+
+def wyraz(s1, s2=""):
+
+    # counts vowels in string
+    def vowels(text):
+        count = 0
+        for letter in text:
+            for a in "aeiouy":
+                if letter is a:
```

Comment:
Powinno być `==` bo `is` to porównanie "czy jest to **ten sam** element" a nie "taki sam"
```suggestion
                if letter == a:
```